python/pysmurf/client/debug/smurf_noise.py

```diff
@@ -955,14 +1008,18 @@ def analyze_noise_vs_bias(self, bias, datafile, channel=None, band=None,
             gs = GridSpec(n_row, n_col)
             ax_NEI = fig.add_subplot(gs[:h_NEI, :w_NEI])
             ax_NEIwl = fig.add_subplot(gs[:h_NEIwl, w_NEI:w_NEI+w_NEIwl])
+
+            # Estimate NEP if IV given
             if est_NEP:
-                if ch not in iv_band_data:
+                if ch not in iv_band_data[b]:
```

Comment:
Same here, name of variable is confusing if it's not the iv data for a single band.